BUILD.md

```diff
@@ -290,46 +219,36 @@ On Linux:
 
 To perform a release build, replace the `assembleDebug` task-name with `assembleRelease`.
 
-#### Building Stand-alone With Android Studio
-
+#### Building with Android Studio
 1. Open Android Studio
-2. Choose "Import Project" and browse to the `{gfxreconstruct_root}/android` folder and click "OK".
- * This option may appear in either the opening Dialog box, or if you choose `File -> New -> Import Project...`
- * Once imported, you should see at least 2 project modules:
-   * layer  - The GFXReconstruct layer that is used to record capture files
-   * replay - The GFXReconstruct capture file replay tool
-3. Build the project
-
-#### Building With Your Existing App and Android Studio
-
-If you are building your app with Android Studio or with gradle, you can do the following to add the layer module as a dependency of your app.
-Adding the layer as a dependency will make gradle build the layer when building your app and add the layer to your app's APK.
-It will also merge the layer module's Manifest file with your app's Manifest file, automatically including the permissions that the layer requires (currently just access to storage).
-
-1. In your app's settings.gradle file, add something like the following:
-
+2. Choose `Import Project`, browse to the `gfxreconstruct/android` folder, and click `OK`.
+  * This option is available through either the opening Dialog box or the `File -> New -> Import Project...` menu item
+  * If import succeeds, the following two project modules will be displayed by the Project browser:
+    * layer  - The GFXReconstruct capture layer for recording Vulkan API data
+    * replay - The GFXReconstruct tool to replay GFXR capture files
+3. Build the project by selecting the `Build -> Make Project` menu item or pressing `Ctrl+F9`
+
+#### Adding the Layer to Existing Projects
+The GFXReconstruct layer project may be added as a dependency of an existing
+Android application project. After the layer has been added as an application
+dependency, it will be built when the application is built and automatically
+included with the application's APK. A manifest file provided by the
+GFXReconstruct layer project will also be merged with the applications Manifest
+file, adding `uses-permission` elements for the permissions required by the
+layer (e.g. WRITE_EXTERNAL_STORAGE).
```

Comment:
for consistency with program constants elsewhere, WRITE_EXTERNAL_STORAGE => `WRITE_EXTERNAL_STORAGE`